libraries/adaptive-expressions/src/converters/objectExpressionConverter.ts

```diff
@@ -9,14 +9,14 @@
 import { ObjectExpression } from '../expressionProperties';
 
 /**
- * `any` value to json `ObjectExpressionConverter` converter.
+ * `any` value to json [ObjectExpressionConverter](xref:adaptive-expressions.ObjectExpressionConverter) converter.
  * @typeparam T The type of the value.
  */
 export class ObjectExpressionConverter<T extends object = {}> {
     /**
-     * Converts value of type `T` into an `ObjectExpression`.
+     * Converts value of type `T` into an [ObjectExpressionConverter](xref:adaptive-expressions.ObjectExpressionConverter).
      * @param value Value of type `T` to convert.
-     * @returns The `ObjectExpression`.
+     * @returns The [ObjectExpressionConverter](xref:adaptive-expressions.ObjectExpressionConverter).
```

Comment:
There is a reason why this is not called ObjectExpression? like before the change

Reply:
good catch